core/src/main/thrift/client.thrift

```diff
@@ -331,6 +338,16 @@ service ClientService {
     1:ThriftTableOperationException tope
   )
 
+  string executeAdminOperation(
+      2:trace.TInfo tinfo
```

Comment:
This can be numbered starting at 1. The numbers for the throws exceptions can also start at 1. They are a separate context, and it doesn't matter if they reuse numbers across contexts, as long as the numbers aren't reused within the same parameter/exception context.